packages/samples/react/src/scenarios/appointment-form/PersonalInformationForm.tsx

```diff
@@ -85,7 +88,10 @@ export function PersonalInformationForm() {
 									id="field-company"
 									_label="Firma"
 									_value={field.value}
-									_error={form.errors.company || ''}
+									_msg={{
```

Comment:
Hi @chrisalat meine Erwartung ist, dass das Property `_error` nach wie vor funktioniert.

Reply:
Wird in der v3 durch _msg ersetzt und in der v2  bereits als deprecated angezeigt. Da _msg  bereits in der v2  funktioniert, dachte ich es wäre besser den Entwicklern direkt das richtige Attribut zu geben, weil sie das Beispiel ja sonst erst mit einem deprecated Wert umsetzen und in der v3 dann wieder anpassen müssen.